docs/src/main/sphinx/connector/faker.md

```diff
@@ -20,83 +23,79 @@ faker.null-probability=0.1
 faker.default-limit=1000
 ```
 
-Create tables in the `default` schema, or create different schemas first.
-Reading from tables in this catalog return random data. See [](faker-usage) for
-more examples.
+Create tables in the `default` schema, or create different schemas first. Tables
+in the catalog only exist as definition and do not hold actual data. Any query
+reading from tables returns random data. As a result, repeated invocation of an
+identical query does not return identical data. See [](faker-usage) for more
+examples.
```

Comment:
The last part is being addressed in https://github.com/trinodb/trino/pull/24008

Reply:
I think we still need a wording here that details this more .. even after 24008 goes in - then we just need to explain that its random and deterministics.. but spell out what that means in practice. E.g. what if you create two identical tables..

Reply:
Agree! The data being deterministic will require explaining it depends on the schema - even with a constant seed, any changes to the schema will affect the order in which generators are called, and may result in getting a completely different data set.